rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -38,567 +38,553 @@ namespace py = pybind11;
 
 using float16 = half_float::half;
 static_assert(sizeof(float16) == 2, "Bad size");
-namespace pybind11
-{
-    namespace detail
-    {
-        constexpr int NPY_FLOAT16 = 23;
-        // Kinda following: https://github.com/pybind/pybind11/blob/9bb3313162c0b856125e481ceece9d8faa567716/include/pybind11/numpy.h#L1000
-        template <> struct npy_format_descriptor<float16>
-        {
-            static constexpr auto name = _("float16");
-            static pybind11::dtype dtype()
-            {
-                handle ptr = npy_api::get().PyArray_DescrFromType_(NPY_FLOAT16);
-                return reinterpret_borrow<pybind11::dtype>(ptr);
-            }
-        };
+namespace pybind11 {
+namespace detail {
+constexpr int NPY_FLOAT16 = 23;
+// Kinda following: https://github.com/pybind/pybind11/blob/9bb3313162c0b856125e481ceece9d8faa567716/include/pybind11/numpy.h#L1000
```

Comment:
@fiona-gladwin Please change language . what does this comment even mean? Remove if not necessary

Reply:
Changed the words